tfupdate/module.go

```diff
@@ -51,12 +53,24 @@ func (u *ModuleUpdater) Update(_ context.Context, _ *ModuleContext, filename str
 	return u.updateModuleBlock(f)
 }
 
+func match(match, matchType, name string) bool {
+	switch matchType {
+	case "regex":
+		var re = regexp.MustCompile(match)
+		return len(re.FindAllString(name, 1)) > 0
```

Comment:
We only need to know if it matches or not; why not use `regexp.MatchString()`?
https://pkg.go.dev/regexp#Regexp.MatchString

Reply:
This is fixed with (https://github.com/minamijoyo/tfupdate/pull/108/commits/e1441c9a5653fb2a577701c7e6000539839dd874)